internal/handlers/handler_sign_u2f_step1.go

```diff
@@ -13,12 +13,14 @@ import (
 
 // SecondFactorU2FSignGet handler for initiating a signing request.
 func SecondFactorU2FSignGet(ctx *middlewares.AutheliaCtx) {
+	// TODO: Should this be operationFailedMessage?
```

Comment:
I would keep it because we don't want the public to know whether X-Forward-... is set or not in the backend...

Reply:
I mean changing it would allow information disclosure for an attacker.

Reply:
@clems4ever I mean, in master we have it set to operationFailedMessage instead of mfaValidationFailedMessage.